eng/jacoco-test-coverage/pom.xml

```diff
@@ -110,37 +110,37 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-common</artifactId>
-      <version>12.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
+      <version>12.1.0</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.1.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-batch</artifactId>
-      <version>12.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
+      <version>12.1.0</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-cryptography</artifactId>
-      <version>12.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-cryptography;current} -->
+      <version>12.1.0</version> <!-- {x-version-update;com.azure:azure-storage-blob-cryptography;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-file-share</artifactId>
-      <version>12.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-storage-file-share;current} -->
+      <version>12.0.0</version> <!-- {x-version-update;com.azure:azure-storage-file-share;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-file-datalake</artifactId>
-      <version>12.0.0-beta.7</version> <!-- {x-version-update;com.azure:azure-storage-file-datalake;current} -->
+      <version>12.0.0-preview.7</version> <!-- {x-version-update;com.azure:azure-storage-file-datalake;current} -->
```

Comment:
Should this be `12.0.0-beta.7`? Preview messes up Maven's most recent release structure. Or is this being done to ensure that preview.7 shows up as the most recent?

Reply:
I'm not really sure of our policy surrounding beta and preview. We've been releasing everything pre-GA as preview, so I thought it would be confusing to suddenly switch to beta, but if there's some engineering systems reason why it needs to be beta now, I don't really feel strongly about it.

Reply:
After GA release we decided to use beta instead of using preview. here is the discussion on this  https://github.com/Azure/azure-sdk/pull/766

Reply:
Plan moving forward is to use `beta` instead of `preview` since Maven doesn't understand `preview` and switches over to a string sort algorithm to determine the most recent release, this is why we saw cases where 12.0.0-preview.5 of Storage Blobs said it was more recent than 12.0.0. Given that this is another preview using preview again will have this show over that but in the future definitely should be using beta.